python/pyspark/sql/column.py

```diff
@@ -200,17 +200,17 @@ class Column:
     ...      [(2, "Alice"), (5, "Bob")], ["age", "name"])
 
     Select a column out of a DataFrame
-
-    >>> df.name
+    # TODO(SPARK-41757): Compatibility of string representation
```

Comment:
Let's put this comment above right before `class Column`. Otherwise, this comment will be shown in PySpark API reference documentation :-).